### PR TITLE
bugfix(TUP-25853) Register dom4j and jaxen

### PR DESCRIPTION
**Pull request by bhe-talendbj**
https://jira.talendforge.org/browse/TUP-25853

### CODE REVIEW

File: main/plugins/org.talend.libraries.dom4j-jaxen/plugin.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>   
+      <extension
+         point="org.talend.core.runtime.librariesNeeded">
+      <libraryNeeded
+              context="plugin:org.talend.libraries.dom4j-jaxen"
+              id="dom4j-2.1.1.jar"
+              mvn_uri="mvn:org.talend.libraries/dom4j-2.1.1/6.0.0"
+              name="dom4j-2.1.1.jar"
```
Comment (wchen-talend) [inline main/plugins/org.talend.libraries.dom4j-jaxen/plugin.xml]: should set the same as component to use :  mvn:org.dom4j/dom4j/2.1.1

File: main/plugins/org.talend.libraries.dom4j-jaxen/plugin.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+              id="dom4j-2.1.1.jar"
+              mvn_uri="mvn:org.talend.libraries/dom4j-2.1.1/6.0.0"
+              name="dom4j-2.1.1.jar"
+              required="true"
+              uripath="platform:/plugin/org.talend.libraries.dom4j-jaxen/lib/dom4j-2.1.1.jar">
+      </libraryNeeded>
+      <libraryNeeded
+              context="plugin:org.talend.libraries.dom4j-jaxen"
+              id="jaxen-1.1.6.jar"
+              mvn_uri="mvn:org.talend.libraries/jaxen-1.1.6/6.0.0"
```
Comment (wchen-talend) [inline main/plugins/org.talend.libraries.dom4j-jaxen/plugin.xml]: mvn:jaxen/jaxen/1.1.6

File: main/plugins/org.talend.libraries.dom4j-jaxen/plugin.xml (hunk 1 above)
Comment (bhe-talendbj) [reply main/plugins/org.talend.libraries.dom4j-jaxen/plugin.xml]: Done

File: main/plugins/org.talend.libraries.dom4j-jaxen/plugin.xml (hunk 2 above)
Comment (bhe-talendbj) [reply main/plugins/org.talend.libraries.dom4j-jaxen/plugin.xml]: Done

**Pull request merged**